animations/test/misc/animated_list_test.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+import 'package:animations/src/misc/animated_list.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createAnimatedListDemoScreen() => MaterialApp(
+      home: AnimatedListDemo(),
+    );
+
+void main() {
+  group('Animated List Tests', () {
+    testWidgets('Initial Length of List', (tester) async {
+      await tester.pumpWidget(createAnimatedListDemoScreen());
+
+      // Get the initial Length of List.
```

Comment:
nit: both "length" and "list" do not need to be capitalized.

(this shows up elsewhere in the file, so it would be a good idea to give the other comments a quick check, too)

Reply:
Sure thing.